scipy/stats/_continuous_distns.py

```diff
@@ -4089,6 +4089,60 @@ def _entropy(self):
 semicircular = semicircular_gen(a=-1.0, b=1.0, name="semicircular")
 
 
+class skew_normal_gen(rv_continuous):
+    """A skew-normal random variable
+    
+    %(before_notes)s
+    
+    Notes
+    -----
+    `skewnormal` takes ``a`` as a skewness parameter
+    When a=0 the distribution is identical to a normal.
+    rvs implements the method of http://azzalini.stat.unipd.it/SN/faq-r.html
+    %(after_notes)s
```

Comment:
* Notes section needs a formula, TeX or pseudocode. See other distributions' docstrings. 

* hyperlinks bitrot. Best give a more solid reference (journal paper, a book, lecture notes, something).

* specific algorithm for drawing random variates is an implementation detail. It can be a second reference, but it anyway needsto be formatted as a reference.

* please add a blank line before the substitution key